legacy/build-deploy-docker-compose.sh

```diff
@@ -615,6 +615,14 @@ if [[ "$BUILD_TYPE" == "pullrequest"  ||  "$BUILD_TYPE" == "branch" ]]; then
     BUILD_ARGS+=(--build-arg ${BUILD_ARG_NAME}="${BUILD_ARG_VALUE}")
   done
 
+  # Here we iterate over any lagoon.base.image data that has been passed to us
+  # in order to explicitly
```

Comment:
Forget to add some words?
```
in order to explicitly pull the images to ensure they are current
```

Reply:
would you believe it if I said I was using "explicitly" ironically :P Leaving everything drastically implicit.

Reply:
my brain hurts now lol